src/fetchtastic/downloader.py

```
@@ -362,6 +378,23 @@ def cleanup_superseded_prereleases(
             prerelease_path = os.path.join(prerelease_dir, dir_name)
+
+            # Check if this is a symlink and remove it for security
+            if os.path.islink(prerelease_path):
+                logger.warning(
+                    "Removing symlink in prerelease dir to prevent traversal: %s",
+                    dir_name,
+                )
+                try:
+                    _safe_rmtree(prerelease_path, prerelease_dir, dir_name)
+                    cleaned_up = True
+                except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)